docs/code-quality/ca1305.md

```diff
@@ -35,6 +35,9 @@ This rule ignores calls to .NET methods that are documented as ignoring the <xre
 - <xref:System.Activator.CreateInstance%2A?displayProperty=nameWithType>
 - <xref:System.Resources.ResourceManager.GetObject%2A?displayProperty=nameWithType>
 - <xref:System.Resources.ResourceManager.GetString%2A?displayProperty=nameWithType>
+- <xref:bool.ToString%2A?displayProperty=nameWithType>
+- <xref:char.ToString%2A?displayProperty=nameWithType>
+- <xref:System.Guid.ToString%2A?displayProperty=nameWithType>
```

Comment:
@Evangelink Can you remind me of the PR on roslyn-analyzers to fix this?

Reply:
https://github.com/dotnet/roslyn-analyzers/pull/3379 and https://github.com/dotnet/roslyn-analyzers/pull/2938

Reply:
@mavasani The changes here look OK to merge?

Reply:
Yes.